packages/web/src/components/notification/Notification/components/icons.tsx

```diff
@@ -78,3 +78,11 @@ export const IconAddTrackToPlaylist = () => {
     <IconPlaylist color='accent' className={styles.iconAddTrackToPlaylist} />
   )
 }
+
+export const IconStreakFire = () => {
```

Comment:
this was the part i was unsure about which pattern to use, but if you're ok with this im ok with it